config/routes.rb

```diff
@@ -1,8 +1,14 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  root to: 'welcome#index'
   get '/privacy', to: 'privacy#index'
-  
-  get '/dashboard', to: 'dashboard#show'
-  
-  resources :gardens, expect: :index
+  get "/users", to: "users#show"
+  resources :user, only: [:show, :update, :destroy]
+  get "/dashboard", to: "dashboard#show"
+  resources :dashboard, only: [:show]
+  resources :gardens, except: [:index]
+  resources :plants, except: [:index]
+  resources :sensors, except: [:index]
+  get "/learn_more", to: "learn_more#show"
+  resources :learn_more, only: [:show]
```

Comment:
More duplicate

Reply:
Good find on all these.  I've updated the routes so there should be no more duplicates